setup.cfg

```diff
@@ -0,0 +1,307 @@
+[metadata]
```

Comment:
So you caught sunpy during its transition to another packaging standard. 

You can delete this file!